homeassistant/components/sensor/zwave.py

```diff
@@ -24,10 +36,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     value.set_change_verified(False)
 
-    # if 1 in groups and (zwave.NETWORK.controller.node_id not in
```

Comment:
This code is to associate devices with the controller. We probably want to put this in a service but let's do it once we pick up Z-Wave more seriously. For now, let's keep it commented out.